lang/en.ts

```diff
@@ -1,6 +1,12 @@
 // @ts-nocheck
 import chalk from 'chalk';
 
+type LangFunction = (...args: string[] | number[]) => string;
```

Comment:
should this be `...args: (string | number)[]`?

Reply:
Yeah actually... you'd think so, but for some reason this works as expected (typing each individual arg as string or number). But I think what you suggested is clearer towards we actually want so I'll update it